src/js/components/Button/__tests__/__snapshots__/Button-test.js.snap

```diff
@@ -778,8 +778,8 @@ exports[`Button hoverIndicator background 1`] = `
 }
 
 .c1:hover {
-  background-color: rgba(221,221,221,0.4);
-  color: #444444;
+  background-color: rgba(0,135,61,0.4);
+  color: #f8f8f8;
   color: #000000;
```

Comment:
Why do we have color twice?

Reply:
not sure, I'll look into it tonight.

Reply:
I suspect this is coming from the `hover` base.js theme object https://github.com/grommet/grommet/blob/4032b2e0e164fa0afc830eb59320089e53ccebf5/src/js/themes/base.js#L219-L229.

Currently, we are using `hover` in `activeStyle` https://github.com/grommet/grommet/blob/ef9432a308f89304908e8f40d56e84c1053cf0af/src/js/utils/background.js#L129
`hover` is a mature object containing the combination of `background-color` and `color`, while each one includes dark/light objects (color  & opacity). (base.js) 
If we are thinking the right approach is to replace `hover` with `active` on `activeStyle` we may need to consider populating an active object in base.js. just a thought. let me know what you think.

Reply:
That is along the lines I am thinking as well.